utils/check-style/aspell-ignore/en/aspell-dict.txt

```diff
@@ -132,6 +132,7 @@ PrettySpace
 PrettySpaceMonoBlock
 PrettySpaceNoEscapes
 PrettySpaceNoEscapesMonoBlock
+PrettyJSONEachRow
```

Comment:
Maybe we should just completely ignore heading `##` and links `[...](...)` in spell checking because lots of ignored items comes from there.